skopt/optimizer/gp.py

```diff
@@ -223,24 +223,11 @@ def gp_minimize(func, dimensions, base_estimator=None,
                     )
 
     space = Space(transformed_dims)
-    # Default GP
-    if base_estimator is None:
-        cov_amplitude = ConstantKernel(1.0, (0.01, 1000.0))
-
-        if is_cat:
-            other_kernel = HammingKernel(
-                length_scale=np.ones(space.transformed_n_dims))
-            acq_optimizer = "sampling"
-        else:
-            other_kernel = Matern(
-                length_scale=np.ones(space.transformed_n_dims),
-                length_scale_bounds=[(0.01, 100)] * space.transformed_n_dims,
-                nu=2.5)
-
-    base_estimator = GaussianProcessRegressor(
-        kernel=cov_amplitude * other_kernel,
-        normalize_y=True, random_state=rng, alpha=0.0, noise=noise,
-        n_restarts_optimizer=2)
+    base_estimator = cook_skopt_estimator(
+        "GP", is_cat=is_cat, random_state=rng, noise=noise,
+        n_dims=space.transformed_n_dims)
+    if is_cat:
+        acq_optimizer = "sampling"
```

Comment:
can we remove this here and leave it to `Optimizer` to correctly select it based on `acq_optimizer='auto'`?